contrib/scalar/scalar.c

```diff
@@ -68,6 +76,24 @@ static void setup_enlistment_directory(int argc, const char **argv,
 	}
 
 	setup_git_directory();
+
+	/* Scalar requires the use of a credential helper */
+	setenv("GIT_TERMINAL_PROMPT", "false", 0);
```

Comment:
This seems wrong.. we're never going to allow terminal-based prompting (including in GCM).
GCM checks `GIT_TERMINAL_PROMPT` too!

Reply:
What I meant to say is that we want to disable Git's own, terminal-based prompt, in favor of credential helpers. Does that sound okay?

Reply:
Maybe this is something we should be setting in the functional tests.

Reply:
Heh, I just had the same idea in https://github.com/microsoft/git/pull/341#discussion_r631008550

Reply:
By setting `GIT_TERMINAL_PROMPT` you are also disabling GCM however.

Reply:
Our functional tests only test against public repos, so that shouldn't be  problem? I don't think GCM Core is even installed for the Linux and macOS builds.